vanilla-framework/vanilla-brochure-theme
# Add .run

## Done
Added `./run` and integrate it into the scripts. 

## QA
- Run `./run` and check the site runs
- Run `./run watch` and check the watcher works
- Run `./run test` and check tests pass
- Run `./run serve` and check the command works
